samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: insert-image
+description: Inserts an image to the current slide.
+script:
+  content: |
+    $('#insert').click(run);
+
+    async function run() {
+        var base64EncodedImageStr = getImageAsBase64String();
+
+        await Office.context.document.setSelectedDataAsync(base64EncodedImageStr, {
+            coercionType: Office.CoercionType.Image,
+            imageLeft: 50,
+            imageTop: 50,
+            imageWidth: 400
+        },
+            function (asyncResult) {
```

Comment:
OK. I am not going to start a discussion here about what is best practice or not.